src/kOS/Suffixed/VesselTarget.cs

```diff
@@ -464,6 +465,13 @@ private void InitializeSuffixes()
             AddSuffix("MESSAGES", new NoArgsSuffix<MessageQueueStructure>(() => GetMessages()));
         }
 
+        public global::Part GetControlPart()
+        {
+            global::Part res = Vessel.GetReferenceTransformPart(); //this can actually be null if "control from here" has never been used
```

Comment:
In a quick test, it looks like this comment is not true.  On freshly launched ship, with the root part set as the 2.5mx3.75m adapter, the suffix returns one of the command pods located on the ship as the value for `CONTROLPART`.  I don't doubt that the value can be null (the internal implementation appears to leave it null if `referenceTransformId == 0u`) but I think I'll revise the comment.  I wanted to give you a chance to comment as well before I simply made the sweeping change.